### DISCUSSION TITLE
Suggest new features

### DISCUSSION THREAD

**Discussion by Tomiwa-Ot** [Tomiwa-Ot/moukthar#28] [Ideas]
Any new ideas?

**Comment by RomanEgovcev** [comment]


**Reply by Tomiwa-Ot** [reply]
Run the command below and send the output
```bash
ss -tulnp
```

**Reply by RomanEgovcev** [reply]
Netid   State    Recv-Q   Send-Q     Local Address:Port      Peer Address:Port  Process                                                                         
udp     UNCONN   0        0          127.0.0.53%lo:53             0.0.0.0:*      users:(("systemd-resolve",pid=495,fd=13))                                      
udp     UNCONN   0        0                0.0.0.0:5353           0.0.0.0:*      users:(("avahi-daemon",pid=639,fd=12))                                         
udp     UNCONN   0        0                0.0.0.0:57815          0.0.0.0:*      users:(("avahi-daemon",pid=639,fd=14))                                         
udp     UNCONN   0        0                   [::]:5353              [::]:*      users:(("avahi-daemon",pid=639,fd=13))                                         
udp     UNCONN   0        0                      *:1716                 *:*      users:(("kdeconnectd",pid=1272,fd=18))                                         
udp     UNCONN   0        0                   [::]:50974             [::]:*      users:(("avahi-daemon",pid=639,fd=15))                                         
tcp     LISTEN   0        128            127.0.0.1:631            0.0.0.0:*      users:(("cupsd",pid=853,fd=7))                                                 
tcp     LISTEN   0        511              0.0.0.0:8080           0.0.0.0:*      users:(("php",pid=1516,fd=5))                                                  
tcp     LISTEN   0        128              0.0.0.0:22             0.0.0.0:*      users:(("sshd",pid=910,fd=3))                                                  
tcp     LISTEN   0        4096       127.0.0.53%lo:53             0.0.0.0:*      users:(("systemd-resolve",pid=495,fd=14))                                      
tcp     LISTEN   0        128                [::1]:631               [::]:*      users:(("cupsd",pid=853,fd=6))                                                 
tcp     LISTEN   0        50                     *:1716                 *:*      users:(("kdeconnectd",pid=1272,fd=19))                                         
tcp     LISTEN   0        511                    *:80                   *:*      users:(("apache2",pid=32316,fd=4),("apache2",pid=32315,fd=4),("apache2",pid=32314,fd=4),("apache2",pid=32309,fd=4),("apache2",pid=32308,fd=4),("apache2",pid=32307,fd=4),("apache2",pid=30642,fd=4),("apache2",pid=30602,fd=4),("apache2",pid=30601,fd=4),("apache2",pid=30600,fd=4),("apache2",pid=30597,fd=4))
tcp     LISTEN   0        128                 [::]:22                [::]:*      users:(("sshd",pid=910,fd=4))                                                  
tcp     LISTEN   0        151                    *:3306                 *:*      users:(("mysqld",pid=30148,fd=28))                                             
tcp     LISTEN   0        2                      *:3389                 *:*      users:(("xrdp",pid=909,fd=11))                                                 
tcp     LISTEN   0        2                  [::1]:3350              [::]:*      users:(("xrdp-sesman",pid=890,fd=7))                                           
tcp     LISTEN   0        70                     *:33060                *:*      users:(("mysqld",pid=30148,fd=21))

**Reply by RomanEgovcev** [reply]
I did it: sudo ufw allow http
My phone is in list But nothing is happening on my phone when I send commands

**Reply by RomanEgovcev** [reply]
[2025-01-07 15:09:45]: 170 => {"web_socket_id":"-1","cmd":"LIST_INSTALLED_APPS","type":"server","referrer":"installed-apps","client":"1"}
[2025-01-07 15:09:45]: Connection closed by 170
For example

**Reply by Tomiwa-Ot** [reply]
Try this
```bash
sudo ufw allow 8080
```
